lib/core/src/lib/login/components/login-dialog.component.stories.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CoreStoryModule } from '../../testing/core.story.module';
+import { RouterTestingModule } from '@angular/router/testing';
+import { LoginModule } from './../login.module';
+import { LoginDialogStorybookComponent } from './login-dialog.stories.component';
+import { MatButtonModule } from '@angular/material/button';
+import { AuthenticationService } from './../../services/authentication.service';
+import { AuthenticationMock } from './../../mock/authentication.service.mock';
+
+export default {
+    component: LoginDialogStorybookComponent,
+    title: 'Core/Login/Login Dialog',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, LoginModule, RouterTestingModule, MatButtonModule],
+            providers: [
+                { provide: AuthenticationService, useClass: AuthenticationMock }
+            ]
+        })
+    ],
+    parameters: {
+        docs: {
+            description: {
+                component: `Allows a user to perform a login via a dialog.`
+            }
+        }
+    },
+    argTypes: {
+        storybookInfo: {
+            control: 'none',
+            description: 'Use `fake-username` and `fake-password` to test functionality.',
```

Comment:
Do we need to add info about the `fake-username-CORS-error`?

Reply:
Added in new commit